### ISSUE TITLE
Is the evaluation code final?

### ISSUE THREAD

**Issue by yuyi1005** [JunMa11/CVPR-MedSegFMCompetition#9]
The evaluation code (3d bbox computation) was updated with this issue https://github.com/JunMa11/CVPR-MedSegFMCompetition/issues/7

I became aware of this inconsistency one month ago, and I optimized my training code to make it work better for bbox from the median voxel (consistent with validation). It improved the accuracy. But with the newly updated validation set, my amendments become useless, even worse than the baseline. 

That's very unfair, especially for those who haven't noticed the update of evaluation code. 

Is there a point where the evaluation code won't be changed anymore? @JunMa11 @kmin940

**Comment by JunMa11** [comment]
Hi @yuyi1005 ,

We're sorry for the inconvenience. 

Before accepting the modification, we first tested the new box with the SAMMed3D model and found it didn't degrade the performance. The DSC score difference was less than 1%.  Thus, we decided to move forward with the revision. 

Sorry again for causing the issue to your solution. We won't have such modification of the evaluation code until the end of the challenge. 

Best regards,
Jun
